components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/SelectOneFunction.java

```diff
@@ -18,12 +18,12 @@
 
 package org.wso2.carbon.identity.application.authentication.framework.handler.sequence.impl;
 
-import org.wso2.carbon.identity.application.authentication.framework.config.model.graph.js.JsAuthenticationContext;
+import org.wso2.carbon.identity.application.authentication.framework.config.model.graph.js.base.JsBaseAuthenticationContext;
 
 /**
  * Functional interface to select one outcome from possible outcomes.
  */
 @FunctionalInterface
 public interface SelectOneFunction {
-    String evaluate(JsAuthenticationContext context, Object possibleOutcomesObj);
+    String evaluate(JsBaseAuthenticationContext context, Object possibleOutcomesObj);
```

Comment:
same as the above

Reply:
Addressed with f00358d0066bbf13b65c87f4867e2ad5052761fc